recorder/src/main/cpp/profiler.cc

```diff
@@ -36,6 +37,7 @@ void Profiler::handle(int signum, siginfo_t *info, void *context) {
         if (thread_info != nullptr) {//TODO: increment a counter here to monitor freq of this, it could be GC thd or compiler-broker etc
             ctx_tracker = &(thread_info->ctx_tracker);
             if (! ctx_tracker->should_record()) {
+                s_c_cpu_samp_drop_norec.inc();
```

Comment:
Probably not required in first-cut. This looks like an unnecessary overhead the first glance specially if some one keeps their coverage low. We can introduce this at a later stage if need be.